packages/studio-ui/src/web/views/FlowBuilder/nodeProps/TransitionSection.jsx

```diff
@@ -86,7 +86,7 @@ export default class TransitionSection extends Component {
       <Fragment>
         <div>
           {items.map((item, i) => (
-            <Fragment>
+            <Fragment key={`${i}.${item.node || '-'}`}>
               <ConditionItem className={style.item} condition={item} position={i} key={`${i}.${item.node || '-'}`}>
```

Comment:
why is the same key used twice ? I thought we only needed it once, also aren't fragments not displayed in the code?

Reply:
My mistake, I should have removed the one on the ConditionItem component. Indeed Fragments are not displayed, but somehow, they can contain a key and their doc also show an example where the key must be on the fragment when mapping a collection (https://reactjs.org/docs/fragments.html#keyed-fragments)